events/artsy-learns-to-code-v2.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Artsy Learns to Code v2
+description: Second iteration on ALTC
+---
+
+See the original ALTC write-up [here](./artsy-learns-to-code.md) for the previous iteration's abstract and
+structure.
+
+## Abstract
+
+Let's kick off another session of Artsy Learns to Code!
+
+## Motivations for ALTC v2
+
+- It's social! Lets Artsy people interact across their normal silos
+- Build empathy across Artsy
+- It's fun! Boosts morale
+- Empowers non-technical people
+- Gets engineers used to teaching regularly
+
+## Features
+
+- Learners can opt to specify different areas of engineering/CS they'd like to learn about. We'll do our best to
+  accomodate these requests, but won't guarantee that we can match mentors' expertises with learners' requests.
+- If students don't have a specific idea for what they'd like to learn (which is totally ok!), we have a default
+  suggested teaching plan: p5.js aka Processing in JS.
+
+## Structure
+
+Similar to the previous ALTC, we'll have trios: one mentor and two learners, who will be partners throughout _Artsy
+Learns to Code_.
+
+As a default course, we'll recommend p5.js, a direct JS port of Processing. We hope that working with p5.js will be
+more engaging than a course on vanilla JS. It also embodies the Artsy value _Art x Science_.
+
+We'll start with this youtube tutorial, which is taught by a professor at NYU's ITP (Interactive Telecommunications
+Program):
+
+https://www.youtube.com/user/shiffman/playlists?view=50&sort=dd&shelf_id=2
+
+In an effort to keep attendance up and keep learners motivated, we won't enforce a schedule that all learners
+should be following. Instead, groups can set their own cadence and work on a schedule that works for them.
+
+Then groups can decide on what they might want to make with p5.js, and mentors can guide them.
+
+### Some Core Competencies We Could Aim Teach
+
+- Fundamentals:
+  - Syntax
+  - Data types & Variables
+  - Conditionals
+  - Functions
+  - Loops
+  - Arrays
+  - Objects
+- Web Development:
+  - What is a library/package?
+  - How does JS/CSS/HTML interact with each other?
+  - What is a server? A client?
+- More General Software Development:
+  - Algorithms
+  - Classes/OOP
+  - SQL
+
+## Expectations of Learners
+
+- Passion to learn and try something new
+- An open mind
+- (Optional) Come with an idea of what they want to learn
+
+## Expectations of Mentoring Engineers
+
+- Mentors should expect to spend 1-2 hrs a week of their time to meet regularly and answer questions async on slack
```

Comment:
For how many weeks?

(For context, trying to balance this time commitment against other time commitments-- like mentoring new engineers! It might be helpful to give a ballpark here.)

Reply:
Great idea, I'll put a ballpark similar to the last ALTC, if not a little shorter. I feel some learners lost some steam near the end, but I'll be sure to note that actual timeline will be at mentor's discretion.